website/docs/docs/build/parallel-batch-execution.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Parallel batch execution"
```

Comment:
Totally in favor of this move, great call. Should we (could we?) include microbatch in the title? "Parallel microbatch execution" so it's clear this isn't related to the larger incremental pages it sits next to but rather the microbatch page?